templates/vue/src/App.vue

```diff
@@ -5,12 +5,12 @@
 </template>
 
 <script>
-import Tapestry from './components/Tapestry'
+import Tapestry from "./components/Tapestry"
 
 export default {
-  name: 'app',
+  name: "app",
   components: {
-    Tapestry,
+    Tapestry
```

Comment:
Is there any way we can keep these commas? Is there a setting for this? The reason is I want to minimize potential merge conflicts with the TYDE branch and this is a culprit of those.